### PR TITLE
Fix deprecation warnings on PHP 8.1

### PR DESCRIPTION
**Pull request by MiRacLe-RPZ**
`strftime` -> `date`
`strlen($null)` -> `!empty($null)`
`dechex($float)` -> `dechex((int)$float)`
remove `is_callable(['classname', 'method'])`

Also some minor fixes and bump staled `Resque::VERSION`

### CODE REVIEW

**Comment by linkuha** [comment]
thanks! I ran into a problem today in the prod when upgrading to 8.2.
the commit [7047123](https://github.com/mjphaynes/php-resque/pull/103/commits/7047123940054372ebd164d007dfc3e12ac30e60) is not only about warnings. due to new case https://www.php.net/manual/en/function.is-callable.php#126199 - now it causes error in my jobs. I need the author to accept this PR

temporary solution is replacing to your fork in composer.json is helped me:
`  
"require": {
    "mjphaynes/php-resque": "php81compat-dev"
},
"repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/MiRacLe-RPZ/php-resque.git"
    }
  ],
`

**Comment by xelan** [comment]
Thanks for your PR @MiRacLe-RPZ and for the comment @linkuha! I'll review and test the changes :+1:

**Comment by xelan** [comment]
Just a few minor comments, could you please have a look @MiRacLe-RPZ?

**Comment by MiRacLe-RPZ** [comment]
Hi, where can i see your comments?

**Comment by xelan** [comment]
> Hi, where can i see your comments?

Hi, in the "Files changed" overview (or here above in the review comments). Thank you!

**Comment by MiRacLe-RPZ** [comment]
nope,
maybe you must Request Changes  for publish comments?

**Comment by xelan** [comment]
Aaah, I'm normally working on Gitlab... done thank you :+1:

File: src/Resque/Job.php [hunk 1]
```diff
@@ -143,8 +143,8 @@ public static function create($queue, $class, array $data = null, $run_at = 0)
     public static function createId($queue, $class, $data = null, $run_at = 0)
     {
         $id = dechex(crc32($queue)).
-            dechex(microtime(true) * 1000).
-            md5(json_encode($class).json_encode($data).$run_at.uniqid('', true));
+            dechex((int)(microtime(true) * 1000)).
+            md5(json_encode($class).json_encode($data).$run_at);
```
[the remark this answers is not in the document]
Comment (MiRacLe-RPZ) [reply src/Resque/Job.php]: uniqid reverted

File: src/Resque/Worker.php [hunk 2]
```diff
@@ -324,7 +324,7 @@ public function work()
                 Event::fire(Event::WORKER_FORK_PARENT, array($this, $job, $this->child));
 
                 $this->log('Forked process to run job on pid:'.$this->child, Logger::DEBUG);
-                $this->updateProcLine('Worker: forked '.$this->child.' at '.strftime('%F %T'));
+                $this->updateProcLine('Worker: forked '.$this->child.' at '.date('Y-m-d H:i:s'));
```
[the remark this answers is not in the document]
Comment (MiRacLe-RPZ) [reply src/Resque/Worker.php:327]: :heavy_check_mark:

**Comment by MiRacLe-RPZ** [comment]
> Just a few minor comments

@xelan, done

**Comment by xelan** [comment]
Thank you very much, @MiRacLe-RPZ

**Pull request merged**